src/OrchardCore.Modules/OrchardCore.Flows/Views/BagPart.Edit.cshtml

```diff
@@ -98,7 +98,7 @@
     {
         Context.Items["BagPart.Edit"] = new object();
         <script asp-name="flowpart-edit" at="Foot"></script>
-        <script asp-name="widgetslist-edit" at="Foot"></script>
```

Comment:
It would be better if the script that uses this declared a dependency on this. Inline `<script>`s can also use `depends-on="widgetslist-edit"`-style dependencies.

Reply:
Thanks for the information about dependencies. Added it to the resource manifest for the style. The script from widgetslist-edit is not needed in this case. It was a regression from #15954.